docs/support.rst

```diff
@@ -2,11 +2,8 @@
 Support
 =======
 
-The easiest way to get help with the project is to open an issue on Github_.
-
-.. The mailing list at ... is also available for support.
-
-.. _Github: https://github.inl.gov/congjian-wang/DACKAR/issues
+The easiest way to get help with the project is to open an issue on GitHub at:
+_Github: https://github.com/idaholab/DACKAR/issues
```

Comment:
there is some issue with the cross reference, please follow the example:
<img width="841" height="106" alt="image" src="https://github.com/user-attachments/assets/3dbb2458-8760-439d-b5db-15790539c5fa" />

The first GitHub need to be GitHub_, and the second one need to be .. _GitHub